src/zope/publisher/tests/test_baserequest.py

```diff
@@ -123,5 +125,6 @@ class ISomeSkin(Interface):
 def test_suite():
     return makeSuite(TestBaseRequest)
 
-if __name__=='__main__':
+
+if __name__ == '__main__':
     main(defaultTest='test_suite')
```

Comment:
TBH we should get rid of these `if __name__ == '__main__':` relics in unit test files.

Maybe there was a day when you could run a single test file with `python src/foo/tests/test_bar.py`, but those days are long gone -- and zope-testrunner has `-m test_bar` for this purpose.

Reply:
Oh, there where plenty of them. I removed them all. 😀